packages/wallets/xdefi/src/xdefiWallet.ts

```diff
@@ -139,7 +139,10 @@ async function getWalletMethodsForChain({
             //@ts-expect-error
             ethereumWindowProvider,
             (
-              toolbox as ReturnType<typeof AVAXToolbox> | ReturnType<typeof BSCToolbox>
+              toolbox as
+                | ReturnType<typeof AVAXToolbox>
+                | ReturnType<typeof BSCToolbox>
+                | ReturnType<typeof ARBToolbox>
```

Comment:
@chillios-dev I was not sure if this is necessary, or if there is a better type here, so that TS knows it has `getNetworkParams`

Reply:
Could do sth like `ReturnType<typeof getToolboxForChain>` but dunno how much cleaner it is 😅 - will check it out